expense_manager/expense_manager/urls.py

```diff
@@ -15,8 +15,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
```

Comment:
Cause you have to fix previous my comment, please remove unused `, include` import(or comment it)

Reply:
done